server/datastore/mysql/software_installers.go

```diff
@@ -196,7 +196,7 @@ INSERT INTO software_installers (
 		id, _ := res.LastInsertId()
 		installerID = uint(id) //nolint:gosec // dismiss G115
 
-		if err := setOrUpdateSoftwareInstallerLabelsDB(ctx, tx, installerID, *payload.ValidatedLabels); err != nil {
+		if err := setOrUpdateSoftwareInstallerLabelsDB(ctx, tx, installerID, *payload.ValidatedLabels, false); err != nil {
```

Comment:
Let's make this a string type (values of either vpp_app_team or software_installer) rather than a bool. That'll also clean up the if below, as you can pass in softwareType as an arg.